packages/app/src/server/routes/index.js

```diff
@@ -189,6 +189,8 @@ module.exports = function(crowi, app) {
   apiV1Router.post('/attachments.remove'               , accessTokenParser , loginRequiredStrictly , csrf, attachment.api.remove);
   apiV1Router.post('/attachments.removeProfileImage'   , accessTokenParser , loginRequiredStrictly , csrf, attachment.api.removeProfileImage);
   apiV1Router.get('/attachments.limit'   , accessTokenParser , loginRequiredStrictly, attachment.api.limit);
+  apiV1Router.post('/attachments.uploadBrandLogo'   , apiLimiter, uploads.single('file'), autoReap, accessTokenParser, loginRequiredStrictly ,csrf, attachment.api.uploadBrandLogo);
+  apiV1Router.post('/attachments.removeBrandLogo'      , apiLimiter, accessTokenParser , loginRequiredStrictly , csrf, attachment.api.removeBrandLogo);
```

Comment:
- `/attachments.*` ではなく、`/_api/v3/customize-setting の中に用意すべきかな
    - jsdoc (swagger) はなしでいい
    - adminRequired をつけないといけない
    - accessTokenParser に対応する必要はない

Reply:
put it in `/_api/v3/customize-setting` instead of /attachments.*.
- No jsdoc (swagger) is needed.
- We need to add `adminRequired`.
- No need to support accessTokenParser